### ISSUE TITLE
coupledvae version release roadmap

### ISSUE THREAD

**Issue by Kevin-Chen0** [Photrek/Coupled-VAE#3]
**coupledvae_v3.1**: Comment the notebook using terms from PDL - TF2 Coursera [course](https://www.coursera.org/learn/probabilistic-deep-learning-with-tensorflow2).

**coupledvae_v4.x**: Rearchitect existing Coupled VAE model with the contents from the PDL - TF2 Coursera course. Including:
- Encoder/decoder
- Gradient Tape
- Display plots per n epochs rather than display after end of training

**coupledvae_v5.x**: Integrate nsc lib.
- v5 will be one where we conduct experimentations for our next paper
